### DISCUSSION TITLE
Changing map types in Livemap mode ?

### DISCUSSION THREAD

**Discussion by tk5ep** [dl9rdz/rdz_ttgo_sonde#292] [Q&A] [answered]
HI,

When using the Livemap mode, I have the choice between the satellite view and the darkmatter view. This later being useless (at least for me).
I didn't find a way to change this in the configurations.

How can I modifiy this, so I have another map type like Mapnik ?

BTW, yesterday I was able to recover 3 sondes with the help of my dl9rdz ttgo receiver ! ;-)

Thanks and 73.

**Comment by tk5ep** [comment]
HI,

I just noticed that when connecting the TTGO with my cellphone, I have the darkmatter map, but when I connect it with my home computer (Windows 10) I get the right standard Openstreetmap !
Any clue what's going on ?

Thanks,
Patrick TK5EP

**Comment by dl9rdz** [comment, marked as answer]
The choice between normal mode and dark mode is made by your browser. So this depends on some settings of the device that you are using to access to live map.

**Reply by tk5ep** [reply]
Thanks Hansi,
Indeed is the browser setting the problem. I tried another browser on my cellphone and this one displays the normal mode.
Now I have to find out what to change..

73 & Thanks
